src/main/java/org/ga4gh/starterkit/wes/controller/Runs.java

```diff
@@ -96,6 +101,7 @@ public RunId createRun(
         @RequestParam(name = "workflow_engine_parameters", required = false) String workflowEngineParameters
         // @RequestParam("workflow_attachment") List<String> workflowAttachment
     ) {
+        loggingUtil.debug("Create run");
```

Comment:
similar to the above. I would add a little more info that helps debug any issues.
Ex: `loggingUtil.debug(String.format("Received a request to create run with params workflow_type = %s, ....",workflowType, workflowTypeVersion....));`

Reply:
This applies to all the logs.